ide-support/revdeploylibraryios.livecodescript

```diff
@@ -1,4 +1,4 @@
-﻿script "revDeployLibraryIOS"
+script "revDeployLibraryIOS"
```

Comment:
Hmm... script only stacks are meant to have a BOM. The engine always writes one when saving.